.buildkite/scripts/publish/build-multiarch-docker.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
should this script be renamed to include a reference to `push`, since it isn't just building, but is also pushing?